org.jacoco.core/src/org/jacoco/core/instr/Instrumenter.java

```diff
@@ -114,7 +114,7 @@ protected String getCommonSuperClass(final String type1,
 				return instrument(new ClassReader(buffer));
 			}
 		} catch (final RuntimeException e) {
-			throw instrumentError(name, e);
+			throw instrumentError("class " + name, e);
```

Comment:
@Godin Why do we need "class" prefix here? Typically the name will be the file name (Example.class) anyways. In case of classes nested in archives this will lead to locations like "archive.jar@class Example.zip". I would prefer not adding a prefix like in `Analyzer`.